tests/models/test_utils.py

```diff
@@ -0,0 +1,82 @@
+import re
+import pytest
+from pydantic import BaseModel
+from optimade.models.utils import OptimadeField, StrictField, SupportLevel
+from typing import List
+
+
+def test_strict_field():
+    """Test :class:`StrictField` creation for failure on bad keys, and
+    warnings with no description.
+
+    """
+    detail = re.escape(
+        "Not creating StrictField ((Ellipsis,), {'random_key': 'disallowed'}) with forbidden keywords ['random_key']."
+    )
+    detail = "forbidden keywords"
```

Comment:
Here (and below) the full warning/error detail wouldn't match, even when using re.escape. Not sure why...